src/jit/compiler.cpp

```diff
@@ -6986,123 +6986,6 @@ void Compiler::TransferTestDataToNode(GenTree* from, GenTree* to)
     }
 }
 
-void Compiler::CopyTestDataToCloneTree(GenTree* from, GenTree* to)
```

Comment:
Huh? Is this really unused? Not sure how I missed it, encountered it in my GT_LIST crusade.

Reply:
I have not found any references since #6102.